code/modules/clothing/suits/cloaks.dm

```diff
@@ -52,6 +62,27 @@
 	allowed = list(/obj/item/weapon/gun/energy/laser/captain)
 	armor = list(melee = 30, bullet = 30, laser = 30, energy = 10, bomb = 25, bio = 10, rad = 10)
 
+/obj/item/clothing/suit/hooded/cloak/goliath
+	name = "goliath hide cloak"
+	icon_state = "golclo"
+	desc = "A cloak made of goliath hide, offers the many benefits of a goliath's hide."
+	allowed = list(/obj/item/device/flashlight,/obj/item/weapon/tank/internals, /obj/item/weapon/resonator, /obj/item/device/mining_scanner, /obj/item/device/t_scanner/adv_mining_scanner, /obj/item/weapon/gun/energy/kinetic_accelerator, /obj/item/weapon/pickaxe, /obj/item/weapon/twohanded/spear)
+	armor = list(melee = 60, bullet = 20, laser = 33, energy = 10, bomb = 70, bio = 60, rad = 20)
+	hooded = 1
+	hoodtype = /obj/item/clothing/head/cloakhood/golhood
+	heat_protection = CHEST|GROIN|LEGS|FEET|ARMS|HANDS
+	max_heat_protection_temperature = FIRE_IMMUNITY_SUIT_MAX_TEMP_PROTECT
+	unacidable = 1
+
+/obj/item/clothing/head/cloakhood/golhood
+	name = "goliath hide hood"
+	icon_state = "golhood"
+	desc = "A stiff hood made of goliath hide."
+	armor = list(melee = 50, bullet = 10, laser = 25, energy = 10, bomb = 30, bio = 40, rad = 10)
```

Comment:
Is that entirely necessary? I mean the head has less material on it and in general head armor in this game is weaker than body armor.